lib/nodeJavaBridge.js

```diff
@@ -21,6 +21,7 @@ java.nativeBindingLocation = binaryPath;
 
 var syncSuffix = undefined;
 var asyncSuffix = undefined;
+var readOnlyFunctionProperties = ['name'];
```

Comment:
I'm sorry, I didn't understand what this was fixing. I see now. You can ignore these two comments.